app/views/work/edit.html.slim

```diff
@@ -21,6 +20,10 @@
         tr
           th =f.label :collection_id, t('.collection')
           td =f.collection_select(:collection_id, @collections, :id, :title)
+        tr
+          th =f.label :document_set_ids, t('.document_sets')
+          td#document_sets
```

Comment:
We add identifier to the html element whose contents we want to replace. In this case, id: 'document_sets'